store/build.gradle

```diff
@@ -1,17 +1,17 @@
 apply plugin: 'java'
-apply plugin: 'me.tatarka.retrolambda'
 
 group = GROUP
 version = VERSION_NAME
 
 dependencies {
-    compile project(path: ':cache')
-    compile libraries.rxJava
-    compile libraries.jsr305
+    implementation project(path: ':cache')
+    implementation libraries.rxJava
```

Comment:
I agree with this one, but not for the middleware comments. RxJava is required to use Store, but even if you use a middleware, you don't really need to know about the parser it uses, because all factories have methods that allow you to create a parser without explicitly using the underlying library.
Also it seems the way `api` works has changed to require `apiElements` instead and alongside the `implementation` directive, so I've done that.